website/pages/docs/commands/operator/raft.mdx

```diff
@@ -0,0 +1,156 @@
+---
+layout: docs
+page_title: operator raft - Command
+sidebar_title: <code>raft</code>
+description: >-
+  The "operator raft" command is used to interact with the integrated Raft storage backend.
+---
+
+# operator raft
+
+This command groups subcommands for operators to manage the integrated Raft storage backend.
+
+```text
+Usage: vault operator raft <subcommand> [options] [args]
+
+ This command groups subcommands for operators interacting with the Vault
+ integrated Raft storage backend. Most users will not need to interact with these
+ commands. Here are a few examples of the Raft operator commands:
+
+Subcommands:
+    join           Joins a node to the Raft cluster
+    list-peers     Returns the Raft peer set
+    remove-peer    Removes a node from the Raft cluster
+    snapshot       Restores and saves snapshots from the Raft cluster
+```
+
+## join
+
+This command is used to join a new node as a peer to the Raft cluster. In order
+to join, there must be at least one existing member of the cluster.
```

Comment:
Can you mention that if using shamir as your unseal mechanism the node will need to be unsealed before it can fully join the cluster?